avae/data.py

```diff
@@ -39,6 +39,12 @@ def load_data(
         print("\nData size:", len(data))
         print("\nClass list:", data.final_classes)
 
+        plot_affinity_matrix(
```

Comment:
I don't think I can actually checkout and run this because this is a fork (correct me if I am wrong and there's a way to do this), but I think this will fail because lookup can be None at this stage (line 29) and there are no checks in the plot_affinity_matrix to make sure it is not None. I suggest moving this function call onto line 27 inside the if statement after the lookup is actually read.

Have you tested this when no affinity matrix is present?

Reply:
Thanks @jolaem you are right. I think I put a check in plot_affinity_matrix. Cant go to line 27 as it needs the classes selected

Reply:
actually just another if condition before line 42

Reply:
Ah, true! Yep, sounds good, add some checks in the plot function then.

Reply:
Actually, instead you could also add ```if lookup is not None:``` just before calling the plot function. I think that might be cleaner